### PR TITLE
JIRA TRAFODION-2731 CodeCleanup: Phase 6. Cleanup obsolete/unused cqds

### PR DESCRIPTION
**Pull request by anoopsharma00**

### CODE REVIEW

File: core/sql/sqlcomp/nadefaults.cpp [hunk 1]
```diff
@@ -503,9 +502,7 @@ SDDkwd__(ALLOW_DP2_ROW_SAMPLING,               "SYSTEM"),
  DDkwd__(CAT_ALLOW_NEW_FEATUREX, "OFF"),
 
 // Control whether authorization caches immutable users
```
Comment (DaveBirdsall) [inline core/sql/sqlcomp/nadefaults.cpp]: Comment should be deleted too.

File: core/sql/sqlcomp/nadefaults.cpp [hunk 2]
```diff
@@ -523,20 +520,14 @@ SDDkwd__(CAT_ENABLE_QUERY_INVALIDATION, "ON"),
 
  // Throw an error if a column is part of the store by clause and
  // is not defined as NOT NULL return an error
```
Comment (DaveBirdsall) [inline core/sql/sqlcomp/nadefaults.cpp]: Comment should be deleted too.

File: core/sql/sqlcomp/nadefaults.cpp [hunk 3]
```diff
@@ -523,20 +520,14 @@ SDDkwd__(CAT_ENABLE_QUERY_INVALIDATION, "ON"),
...
- DDkwd__(CAT_ERROR_ON_NOTNULL_STOREBY, "ON"),
 
- DDui1__(CAT_FS_TIMEOUT,                       "9000"),
 
  // Used to make ignore "already exists" error in Create and
  // "does not exist" error in Drop.
```
Comment (DaveBirdsall) [inline core/sql/sqlcomp/nadefaults.cpp]: Comment should be deleted too.

File: core/sql/sqlcomp/nadefaults.cpp [hunk 4]
```diff
@@ -523,20 +520,14 @@ SDDkwd__(CAT_ENABLE_QUERY_INVALIDATION, "ON"),
...
- DDkwd__(CAT_IGNORE_ALREADY_EXISTS_ERROR, "OFF"),
- DDkwd__(CAT_IGNORE_DOES_NOT_EXIST_ERROR, "OFF"),
 
  // Used to make catman test134 predictable
```
Comment (DaveBirdsall) [inline core/sql/sqlcomp/nadefaults.cpp]: And here

File: core/sql/sqlcomp/nadefaults.cpp [hunk 5]
```diff
@@ -523,20 +520,14 @@ SDDkwd__(CAT_ENABLE_QUERY_INVALIDATION, "ON"),
...
- DDkwd__(CAT_IGNORE_EMPTY_CATALOGS, "OFF"),
 
  // Catalog Manager internal support for REPLICATE AUTHORIZATION
```
Comment (DaveBirdsall) [inline core/sql/sqlcomp/nadefaults.cpp]: And here

File: core/sql/sqlcomp/nadefaults.cpp [hunk 6]
```diff
@@ -523,20 +520,14 @@ SDDkwd__(CAT_ENABLE_QUERY_INVALIDATION, "ON"),
...
- DDkwd__(CAT_IGNORE_REPL_AUTHIDS_ERROR, "OFF"),
 
 // This enables the DB Limits functionality.  If set to OFF, then blocksize
 // is restricted to 4096 and clustering key size is limited to 255 bytes.
```
Comment (DaveBirdsall) [inline core/sql/sqlcomp/nadefaults.cpp]: And above

File: core/sql/sqlcomp/nadefaults.cpp [hunk 7]
```diff
@@ -547,40 +538,29 @@ SDDkwd__(CAT_ENABLE_QUERY_INVALIDATION, "ON"),
 // If DB Limits is enabled, then increase the default blocksize to 32K
 // on NSK if the object's clustering key length is larger than this value.
```
Comment (DaveBirdsall) [inline core/sql/sqlcomp/nadefaults.cpp]: And here

File: core/sql/sqlcomp/nadefaults.cpp [hunk 8]
```diff
@@ -547,40 +538,29 @@ SDDkwd__(CAT_ENABLE_QUERY_INVALIDATION, "ON"),
...
 
-  DDui1__(CAT_LARGE_BLOCKS_MAX_KEYSIZE,   "1"),
 
 // If DB Limits is enabled, then increase the default blocksize to 32K
 // on NSK if the object's row size is larger than this value.
```
Comment (DaveBirdsall) [inline core/sql/sqlcomp/nadefaults.cpp]: And here

File: core/sql/sqlcomp/nadefaults.cpp [hunk 9]
```diff
@@ -547,40 +538,29 @@ SDDkwd__(CAT_ENABLE_QUERY_INVALIDATION, "ON"),
...
 // Controls how pathnames for routines/procedures/SPJs are interpreted
   DDkwd__(CAT_LIBRARY_PATH_RELATIVE, "OFF"),
 
-  DDkwd__(CAT_MORE_SCHEMA_PRIVS, "ON"),
 
-  DDkwd__(CAT_OVERRIDE_CREATE_DISABLE, "OFF"),
 
 
 // This forces an rcb to be created with a different version number
 // A "0" means to take the current mxv version
```
Comment (DaveBirdsall) [inline core/sql/sqlcomp/nadefaults.cpp]: And in numerous additional places

**Comment by anoopsharma00** [comment]
At  high level, this was done in multiple steps:
- use showcontrol to find out all cqds
- find out which file and code location they are used using greps etc
- insert the cqd and location (null if  not found) in our favorite database (Trafodion). 
- run queries to  find out cqd usage (how often it is used, top used cqds, total cqds,  unused cqds, etc)
- delete unused ones from DefaultConstants and nadefaults using another script.
  (some manual deletion of code comments is needed as you found out during your review).

Will clean up the scripts and deliver them in a subsequent checkin.

The next step in cqd removal is to look at cqds and where they are used. And then determine if
the code using them is obsolete and should also be removed.

**Pull request merged**